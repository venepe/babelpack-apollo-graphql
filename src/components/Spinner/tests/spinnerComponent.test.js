import React from 'react'
import { shallow } from 'enzyme'
import SpinnerComponentTest from '../index'


describe('SpinnerComponentTest Component', () => {
  it('renders with all props', () => {
    const wrapper = shallow(
      <SpinnerComponentTest />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
