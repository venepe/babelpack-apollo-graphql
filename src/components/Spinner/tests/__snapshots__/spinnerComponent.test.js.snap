// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerComponentTest Component renders with all props 1`] = `
<styled.div>
  <div />
  <div />
  <div />
</styled.div>
`;
